test/blocks/merch/merch.test.js

```diff
@@ -1,266 +1,268 @@
 import { readFile } from '@web/test-runner-commands';
 import { expect } from '@esm-bundle/chai';
-import sinon from 'sinon';
-import { createTag, setConfig } from '../../../libs/utils/utils.js';
 
-const config = setConfig({ codeRoot: '/libs', env: { name: 'local' } });
-const {
-  default: merch,
-  VERSION,
-  getTacocatEnv,
-  getTacocatLocale,
-  getTacocatMetadata,
-  imsCountryPromise,
-  runTacocat,
-} = await import('../../../libs/blocks/merch/merch.js');
+import { mockIms, unmockIms } from './mocks/ims.js';
+import { mockWcs, unmockWcs } from './mocks/wcs.js';
+import { createTag, setConfig } from '../../../libs/utils/utils.js';
 
-document.head.innerHTML = await readFile({ path: './mocks/head.html' });
-document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const config = { codeRoot: '/libs', env: { name: 'prod' } };
 
 describe('Merch Block', () => {
+  let Merch;
+
+  after(async () => {
+    delete window.lana;
+    unmockWcs();
+    unmockIms();
+  });
+
   before(async () => {
-    Object.assign(window.tacocat, {
-      loadPromise: Promise.resolve(false),
-      price: { optionProviders: [] },
-      defaults: {
-        apiKey: 'wcms-commerce-ims-ro-user-milo',
-        baseUrl: 'https://wcs.stage.adobe.com',
-        landscape: null,
-        env: 'STAGE',
-        environment: 'STAGE',
-        country: 'US',
-        clientId: 'adobe_com',
-        language: 'en',
-        locale: 'en_US',
-        checkoutWorkflow: 'UCv3',
-        checkoutWorkflowStep: 'email',
-      },
-    });
+    window.lana = { log: () => {} };
+    document.head.innerHTML = await readFile({ path: './mocks/head.html' });
+    document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+    await mockIms('CH');
+    await mockWcs();
+    setConfig(config);
   });
 
-  it('Doesnt decorate merch with bad content', async () => {
+  beforeEach(async () => {
+    const { init, Log } = await import('../../../libs/deps/commerce.js');
+    await init(() => config);
+    Log.reset();
+    Log.use(Log.quietFilter);
+    Merch = (await import('../../../libs/blocks/merch/merch.js')).default;
```

Comment:
merch being a block, we use lower cases, why capitalize first letter?

Reply:
renamed